log4j-core/src/main/java/org/apache/logging/log4j/core/config/properties/PropertiesConfigurationBuilder.java

```diff
@@ -190,6 +193,17 @@ public PropertiesConfiguration build() {
         return builder.build(false);
     }
 
+    private static void useSyntheticLevelAndAppenderRefs(final String propertyName, final Properties loggerProps, final Properties context) {
+        final String propertyValue = (String) context.remove(propertyName);
+        if (propertyValue != null) {
+            final String[] parts = propertyValue.split(",");
+            loggerProps.setProperty("level", parts[0]);
+            for (int i = 1; i < parts.length; i++) {
+                loggerProps.setProperty("appenderRef.$" + i + ".ref", parts[i]);
```

Comment:
It may be worthwhile to add some StatusLogger debug logging here, I defer to your judgement :-)

Reply:
Good idea; added.